main.js

```diff
@@ -1,48 +1,75 @@
 import * as THREE from 'three';
```

Comment:
ooohhh Three.js 👀

Never used it myself, but definitely heard a lot about it.

Also, looks like you've got imports and npm working, great job!